### ISSUE TITLE
Save relative paths from the config module

### ISSUE THREAD

**Issue by stsewd** [rtfd/readthedocs.org#5364]
We currently are saving the whole path, including the root of the servers, that info isn't useful for us or for the user later. We will not lose any kind of information, because we already have the base path to validate files. This shouldn't be a big refactor, and we have a lot of tests.

Ref https://github.com/rtfd/readthedocs.org/pull/4863#discussion_r260421353

**Comment by humitos** [comment]
I saw that we are using absolute path in other places, not just in this config module. For example, on the build output page --which generate to scroll horizontally to know the command executed.

Do you think it worth to consider to user relative path in other places also? What are your thoughts on this?

**Comment by stsewd** [comment]
> Do you think it worth to consider to user relative path in other places also? What are your thoughts on this?

Yeah, when I refactored the code, I tried to put most of the commands with relative paths (like requirements, conda env, etc).

We still depend on absolute urls to set the cache to pip, virtualenv root, python binnary.

We can remove some of those using env variable, after all, isn't helpfull at all for the user to know where we are saving the cache/virtualenv. This is another issue, not in the scope of this one.
